CHANGELOG-1.x.md

```diff
@@ -1,3 +1,7 @@
+## 4.3.0 (2022-04-20)
```

Comment:
Not a huge deal but since we need to update the tag below the date is technically 4-22 so we could update this at the same time